### PR TITLE
Forbid rule-abiding robots from indexing.

### PR DESCRIPTION
**Pull request by MartinLeedotOrg**
Resolves github issue #4257

@juliusv

### CODE REVIEW

File: web/ui/templates/_base.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name=”robots” content=”noindex,nofollow”>
```
Comment (juliusv) [inline web/ui/templates/_base.html]: Thanks! Could you use the regular `"` quotes here instead of `”`?

Comment (MartinLeedotOrg) [reply web/ui/templates/_base.html]: Amended. Not sure how you want to operate in this repo, should I rebase?

**Comment by brian-brazil** [comment]
We can squash ourselves on merge. This is not a bugfix, so given the current moratorium on non-bugfix changes this will not be merged for 2 weeks.

**Comment by MartinLeedotOrg** [comment]
Understood!

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**